pkgs/by-name/wl/wl-gammarelay-rs/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  stdenv,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "wl-gammarelay-rs";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "maxverevkin";
```

No reason to not use proper capitalisation here.
```suggestion
    owner = "MaxVerevkin";
```